globals/utilities.php

```diff
@@ -166,12 +166,17 @@ function neve_hooks() {
  *
  * @return string|null
  */
-function neve_cart_icon( $echo = false, $size = 15, $cart_icon = '' ) {
+function neve_cart_icon( $echo = false, $size = 15, $cart_icon = '', $icon_custom = '' ) {
 	$icon = '<svg width="' . $size . '" height="' . $size . '" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M704 1536q0 52-38 90t-90 38-90-38-38-90 38-90 90-38 90 38 38 90zm896 0q0 52-38 90t-90 38-90-38-38-90 38-90 90-38 90 38 38 90zm128-1088v512q0 24-16.5 42.5t-40.5 21.5l-1044 122q13 60 13 70 0 16-24 64h920q26 0 45 19t19 45-19 45-45 19h-1024q-26 0-45-19t-19-45q0-11 8-31.5t16-36 21.5-40 15.5-29.5l-177-823h-204q-26 0-45-19t-19-45 19-45 45-19h256q16 0 28.5 6.5t19.5 15.5 13 24.5 8 26 5.5 29.5 4.5 26h1201q26 0 45 19t19 45z"/></svg>';
 	if ( ! empty( $cart_icon ) && class_exists( '\Neve_Pro\Modules\Header_Footer_Grid\Components\Icons' ) ) {
 		$cart_icon_svg = Icons::get_instance()->get_single_icon( $cart_icon );
 		$icon          = ! empty( $cart_icon_svg ) ? $cart_icon_svg : $icon;
 	}
+
+	if ( $cart_icon === 'custom' ) {
+		$icon = $icon_custom;
+	}
+
```

Comment:
Maybe I'm missing something but shouldn't the `sanitize_callback` of the setting be enough to escape the custom code?

Reply:
You are right indeed, but that would only sanitize the content that is saved to the DB on the customizer publish. 

You could filter the theme mod value to anything else and skip that sanitization when outputting to the front end. A common example of this happening is translation plugins that can translate all the customizer settings 😄.

It is important to escape all the data echoed as late as possible due to the fact that this could be always changed. 🚀 

I hope this makes sense! Let me know if you have other questions 🦭

Reply:
That's great :smile: Thank you! :rocket: